ingestion-core/src/main/java/com/mozilla/telemetry/ingestion/core/util/Json.java

```diff
@@ -39,7 +39,7 @@ public static ObjectNode asObjectNode(Map<String, ?> map) {
    * Use {@code MAPPER} to convert {@link ObjectNode} to {@code Map<String, Object>}.
    */
   public static Map<String, Object> asMap(ObjectNode objectNode) {
-    return MAPPER.convertValue(objectNode, new TypeReference<Object>() {
+    return MAPPER.convertValue(objectNode, new TypeReference<Map<String, Object>>() {
```

Comment:
@relud I missed this in the last review, but `TypeReference<Object>()` looks like a mistake here. Does this change make sense to you?

Reply:
lgtm